<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ValveSequenceRegistration</class>
 <widget class="qMRMLWidget" name="ValveSequenceRegistration">
  <property name="enabled">
   <bool>false</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>557</width>
    <height>118</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="horizontalSpacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="timePointLayout">
     <item>
      <widget class="QPushButton" name="addTimePointButton">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cardiacCyclePhaseSelector">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removeTimePointButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <widget class="qMRMLSequenceBrowserPlayWidget" name="heartValveBrowserPlayWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="RecordingControlsVisible">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="qMRMLSequenceBrowserSeekWidget" name="heartValveBrowserSeekWidget"/>
     </item>
     <item>
      <widget class="QLabel" name="valveVolumeSequenceIndexLabel">
       <property name="text">
        <string>Volume index: NA</string>
       </property>
       <property name="margin">
        <number>6</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="goToAnalyzedFrameButton">
       <property name="text">
        <string>Go to</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qMRMLSequenceBrowserPlayWidget</class>
   <extends>qMRMLWidget</extends>
   <header>qMRMLSequenceBrowserPlayWidget.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLSequenceBrowserSeekWidget</class>
   <extends>qMRMLWidget</extends>
   <header>qMRMLSequenceBrowserSeekWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>ValveSequenceRegistration</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>heartValveBrowserPlayWidget</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>278</x>
     <y>155</y>
    </hint>
    <hint type="destinationlabel">
     <x>317</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ValveSequenceRegistration</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>heartValveBrowserSeekWidget</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>278</x>
     <y>155</y>
    </hint>
    <hint type="destinationlabel">
     <x>226</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
